diff -uprN -X linux-5.8.1/Documentation/dontdiff linux-5.8.1/arch/x86/entry/syscalls/syscall_64.tbl linux-5.8.1Navneel.0360/arch/x86/entry/syscalls/syscall_64.tbl
--- linux-5.8.1/arch/x86/entry/syscalls/syscall_64.tbl	2020-08-11 19:18:12.000000000 +0530
+++ linux-5.8.1Navneel.0360/arch/x86/entry/syscalls/syscall_64.tbl	2021-02-14 20:50:36.971869448 +0530
@@ -403,3 +403,8 @@
 545	x32	execveat		compat_sys_execveat
 546	x32	preadv2			compat_sys_preadv64v2
 547	x32	pwritev2		compat_sys_pwritev64v2
+
+#
+# My own syscall
+#
+548 64 mySyscall        sys_mySyscall
diff -uprN -X linux-5.8.1/Documentation/dontdiff linux-5.8.1/certs/signing_key.pem linux-5.8.1Navneel.0360/certs/signing_key.pem
--- linux-5.8.1/certs/signing_key.pem	1970-01-01 05:30:00.000000000 +0530
+++ linux-5.8.1Navneel.0360/certs/signing_key.pem	2021-02-14 21:58:50.785587579 +0530
@@ -0,0 +1,83 @@
+-----BEGIN PRIVATE KEY-----
+MIIJQwIBADANBgkqhkiG9w0BAQEFAASCCS0wggkpAgEAAoICAQCm3+Yu6hjzAyZX
+5+pS25OmU6wZxW3bT+RVQOJfIKx3hCIn9C7Qf5N2seKu3wNTZJYXxGQtkbeoLHFa
+hrZhAl9hBArNnLj1MWAvbofLdGYfaJtui8HEq/6gUCSTLwvmPxMAgAxey8Rahcb0
+dmhwOHDRyn+Uc9nLS+IagQojaAQiI0eaJLrBeqh9wyVxTB9S4CAfE86crU5wi4JS
+GDJfK+XAyOE5IkFDVnA6hF35lBENpc8d7M62W98nzaXzxWJTIuoeUBbrFjBzk3Io
+Rt3dLX8b6PLaZ4ivj43cZwQyXeMnf+y95FRYY5HCIJGn9kz0kZQ+Mu/+19VlCnbq
+sIXv0YnbvhUDO70UHj6Zfr2BzH2RB2uTvFSUryYxg2C2Q3lcF1lmn/sDOzZglmBe
++7B0MQzIm9o0A7/16ZrDtu1dJJsZKRaYhpPqjZgKez8aKJAO3odHHNEaPv3+wDrQ
+7OwELe2fVQNTYAvE6Zj4Uv8BvekAEvmhDR+o1SWgkcJTDAtUiZz02tVLtU1a0ZYc
+6WcfXR8KPFkzv/PpvvctmiIZygRNqEDDjjVbk2vouEESywHza/oHtFY+nQiEhaiE
+QKXfXrLEdwkuw3Ew4h6EAX3GfRcs19y++dlk4JXiI+vVqz6Y3zSb3xnrmMMw/YNM
+PH/V5WvktXcMySBsn3ofZOtOxHu8qQIDAQABAoICADyK5PdGZXDdRq8D7AbNtT7W
+Ao2DDH1R+fRYSwjaUNg3rzCxrb5lWrnoheaN8KXzy2SqEvJZeLliDmoaj9aYRsRq
+9ARvBMeod04eGtRc0aQ5jpLHz7Ar+oBtMyfXHMkXPyyiSFqoMP/qoykUQIXeZEgo
+4KqacX0JdenJwUThOwarqBGrm7oQZ0rhXxdA1+h4eg7sX7yGP/wjEU/5iWTwBzYv
+a08QN39Xv/XrCjE/YASEkKU5wjF1GXxWZwwPI8In26rNwdLexQtZhkRImbviAsi8
+dkd44qVXPs+k7Gfdmkk9Z0XprhQ3Y617pXy7pmXg31jFoD7kTX6QEl7JVn6QliP2
+AwkBvb6HUMv23YfWeCi5RywR04MVhhalrMLEElwI3s+qp15dqzwB6QSzeJkS4Tvn
+NjpMy/SVn5rZ13AmrBxisefCnHYV1gHMeuuCU/fxQ7r3/5UnTYG9uUIZeY/qaw7Z
+3Su9uugefRPxeO/mBvuyb2rd0HdiEytQQ1LRLTABurtTcdnTFP97/mwepNJnV7YW
+tvBKcmYcIFN9Up/EeLDEjcMFK9qycXpnKhjlL3neDuA0SP5bYbdXv4Wllm4dk9ww
+tvEU94d7jyxrDSXOUGZ6juysUWVMH8ZvAfW3xLJvgzh7zZrRAeY7ebl50N5xU18Q
+s6g+PbO8T1V2qaHt4zhFAoIBAQDciEeRjYJjHbtg4/WxtaDHLHn1mjebYdIem132
+hjjbKcqu7oXFHwtmylb6bhACDddX3fdFLR8E7W+qHMSPMcMcpntdJnpmbROo0xsO
+qEVKivwyt/wA02wTN8cFnu0Ijf0Ho5mROhmahmk5cNofPI9UwJT22mqb4t6ZPSPe
+6/QvekCiVkREDp/0JgA/78h1Vu8hNerIPxMj7hn6m5tlKKYCK4S9MDRldpdh7OYg
+PTzkDNHNmVMsrbpCK9NAIscwvHWb2E3eA+4NzZl9XRLLUQ9Gkw4OyoAelw8Bjp8g
+MosFDIvDsTBVepKdTVT9YhfpFuoqLBHs9J3mBZZqm32e85pTAoIBAQDBtm6sqjL7
+uaIpEvrnoN/yCcp9ghOCBu46Meago1Vqj1HDxj90lo4/dLZmFhodP7/lOeHGuKMh
+qFwZR5F3NUHG50ZhyfAqe6Sc51GHR31hK5EVgOhVzYmQkPThBx0GWt9JJHLVtqg5
+sAy7CSDLSsdQSySOpkJ+AtIggD3TCZfl/3bF78Dr0K1lW5xlwUYhDG1AW6jRx1Ra
+530Y+74Arf2eSD24EHLghbCuJb9vjuwGZSYa1f/LcLFKZpgIZzvArimu8DdBxuvd
+n0mE5ZggbpAX69MOi2SmYMY0rI5ONfH20jhSCdZ8OJhf7mvrZrzU37xdl1lJAGg8
+a7Ky5SqXP4WTAoIBAQCyOxnMUte8Dz1vEw1wsF/eWN3AVwD1/Vr6DYslQFkXF3GN
+a4yFQY/PZVxiTS1224TcqLG9ILhfHR4QF+vr5sXPk+VD5WMrAKDQ4ryNXgWSNz2Y
+KAXtDPHdnkilDEp5jXegAuKL8q7aqiLEn6ZJdIJKdtJtpxxeHD7XDgi9k2joLh+q
+k5ncfw1bHpM5GSujWb0t1AuKByAK+9XXKoeue+me+1juJdVS40N5iyJ5j8R0cr/l
+MDN+enF8JTs4LGNbyhQq/egAtKBiW3XfUads1nhIC3VjEtn03xTrpz6HoRGl5PvY
+Do4Hp2Zy5xr3JZ45baGWr8NVBGXQj1d/lWczXaGtAoIBAA/UxP6/eiMwbJxGvNKp
+RyQUwMO5GbcKsfV3lo2SS6zFwbzqOs1gcIenFjEJpWC2kQw+ysDe3hYmGEDIpI7F
+f3at+AaN5l7SdAWOsg8IPKrwUI8dVd+NSKzT6+9oEWsviggEPo5QuzbbuG6gtFlo
+5xkcywQNemmp+VpGoMbahdt6IU7T5URgbYtXDVUTkpOo3FTiKM3cKHy1ZnfgOwr7
+LTr+VGzyizDmPCY/V4HagFeVdKPE0X7i2tplwA8PwMZidW2inCF4DFa0aeG+kcw0
+vPn3f+w0YClox0oOwCKMLhYp4Ah0XV8RkQJmDdPTNaOAFXjgoZ9htSMa+iO8rtpl
+ipkCggEBALe8aHOiKHYulHGIwaU1njBVIDMxon0aK1wObtfNSN8VdHvQu+qmZM/X
+0wMDKLv2XVtgRLFG7XlfhrWP9+PeZd7G2nGK7ZQyHUaasAJK/R6ss4sJjL9WEcJE
+e0odaTg3EOGHagRrXzpLFKgBI1FgEYYEfguI7qBtidpAf2J8S9wQzEPMVnTjscsU
+JFGeMVTJ5xCLV7VijdQVXvnzUFPpd4r9ix5V8KlolIWbbEHtuqUW/p8snoJMiM0P
+bEZyALOG4tuMN5++m+1NlO7yz5ucp/RBaYsz05KQ1HOZx5hZCgBEYQmd3oPBL20s
+iBrMeLElYR73xjah31kfotbqLJKJqN0=
+-----END PRIVATE KEY-----
+-----BEGIN CERTIFICATE-----
+MIIFSTCCAzGgAwIBAgIUGwA4T6Wk89izralBV7rmEM52/yswDQYJKoZIhvcNAQEN
+BQAwLjEsMCoGA1UEAwwjQnVpbGQgdGltZSBhdXRvZ2VuZXJhdGVkIGtlcm5lbCBr
+ZXkwIBcNMjEwMjE0MTYyODUwWhgPMjEyMTAxMjExNjI4NTBaMC4xLDAqBgNVBAMM
+I0J1aWxkIHRpbWUgYXV0b2dlbmVyYXRlZCBrZXJuZWwga2V5MIICIjANBgkqhkiG
+9w0BAQEFAAOCAg8AMIICCgKCAgEApt/mLuoY8wMmV+fqUtuTplOsGcVt20/kVUDi
+XyCsd4QiJ/Qu0H+TdrHirt8DU2SWF8RkLZG3qCxxWoa2YQJfYQQKzZy49TFgL26H
+y3RmH2ibbovBxKv+oFAkky8L5j8TAIAMXsvEWoXG9HZocDhw0cp/lHPZy0viGoEK
+I2gEIiNHmiS6wXqofcMlcUwfUuAgHxPOnK1OcIuCUhgyXyvlwMjhOSJBQ1ZwOoRd
++ZQRDaXPHezOtlvfJ82l88ViUyLqHlAW6xYwc5NyKEbd3S1/G+jy2meIr4+N3GcE
+Ml3jJ3/sveRUWGORwiCRp/ZM9JGUPjLv/tfVZQp26rCF79GJ274VAzu9FB4+mX69
+gcx9kQdrk7xUlK8mMYNgtkN5XBdZZp/7Azs2YJZgXvuwdDEMyJvaNAO/9emaw7bt
+XSSbGSkWmIaT6o2YCns/GiiQDt6HRxzRGj79/sA60OzsBC3tn1UDU2ALxOmY+FL/
+Ab3pABL5oQ0fqNUloJHCUwwLVImc9NrVS7VNWtGWHOlnH10fCjxZM7/z6b73LZoi
+GcoETahAw441W5Nr6LhBEssB82v6B7RWPp0IhIWohECl316yxHcJLsNxMOIehAF9
+xn0XLNfcvvnZZOCV4iPr1as+mN80m98Z65jDMP2DTDx/1eVr5LV3DMkgbJ96H2Tr
+TsR7vKkCAwEAAaNdMFswDAYDVR0TAQH/BAIwADALBgNVHQ8EBAMCB4AwHQYDVR0O
+BBYEFBa7GQjrVi9ghhMfL7C2PDufRdcoMB8GA1UdIwQYMBaAFBa7GQjrVi9ghhMf
+L7C2PDufRdcoMA0GCSqGSIb3DQEBDQUAA4ICAQBe/e33p2utbb2mPrqYhZr3x3BR
+I0shdC6mZeeG/cQCviZ61O4aELdTFpLyistARkIjuqBUFXYVBna7n9O4qEcNp/0k
+/3ClZwHiVTwjGNIoyL/7uOoi61REDRv1CYxWxtUm12Qic0Ha5FnH5QUHzG5ARdxM
+j6m9eRexoClOiN/SzVmdAPnxAPnx9YpPhKO7FiY3jiIOvgnpNNVSDwETQdk/xV9j
+odscmFARVm5thNuJhO7rbjOviH9ugkinvSkPmK9A0ETNud6CgWhKCKRa1HdPyR+Y
+uFoOVhJwODWj5/z2vIsyvChIb2yCeqBuI+Qmwr7q7LRAQwpMoqWOWeuKepBE33Oq
+VA0sjhvzn2mG7N7o0Qnrd+T+j92Soi+SIZY8NQMZOG40ETLjCjKUadeK5SUx0l6u
+hxfpy8rcGPaiDge1ZuH0fO6w+qZ08NVX5pz6tWMg7zvdrGfj1lLBUF1jB8lwJslD
+tFpk+p2zq+uCQiI8OyEKQk0fLrtt1wXphgxxCT9ypFSyNTg5fAbRRTX2SCsCT4zS
+spbzkpwfcGjVvN6P4CZG3x2hcMM63AfXTjDO6M/IVeQ8Ssc/XoT5nHjAEQkKJ6oi
+BKjjFQLaik4i8mPFCMj+qv8sd4Lj/lD3Ik3tZw0l0n9ovnphX62zztR3aKuz2ptU
+bh53FUruna9fL5m4oQ==
+-----END CERTIFICATE-----
Binary files linux-5.8.1/certs/signing_key.x509 and linux-5.8.1Navneel.0360/certs/signing_key.x509 differ
diff -uprN -X linux-5.8.1/Documentation/dontdiff linux-5.8.1/certs/x509.genkey linux-5.8.1Navneel.0360/certs/x509.genkey
--- linux-5.8.1/certs/x509.genkey	1970-01-01 05:30:00.000000000 +0530
+++ linux-5.8.1Navneel.0360/certs/x509.genkey	2021-02-14 21:58:48.321546094 +0530
@@ -0,0 +1,17 @@
+[ req ]
+default_bits = 4096
+distinguished_name = req_distinguished_name
+prompt = no
+string_mask = utf8only
+x509_extensions = myexts
+
+[ req_distinguished_name ]
+#O = Unspecified company
+CN = Build time autogenerated kernel key
+#emailAddress = unspecified.user@unspecified.company
+
+[ myexts ]
+basicConstraints=critical,CA:FALSE
+keyUsage=digitalSignature
+subjectKeyIdentifier=hash
+authorityKeyIdentifier=keyid
diff -uprN -X linux-5.8.1/Documentation/dontdiff linux-5.8.1/Makefile linux-5.8.1Navneel.0360/Makefile
--- linux-5.8.1/Makefile	2020-08-11 19:18:12.000000000 +0530
+++ linux-5.8.1Navneel.0360/Makefile	2021-02-14 19:13:48.418775818 +0530
@@ -1070,7 +1070,7 @@ export MODORDER := $(extmod-prefix)modul
 export MODULES_NSDEPS := $(extmod-prefix)modules.nsdeps
 
 ifeq ($(KBUILD_EXTMOD),)
-core-y		+= kernel/ certs/ mm/ fs/ ipc/ security/ crypto/ block/
+core-y		+= kernel/ certs/ mm/ fs/ ipc/ security/ crypto/ block/ mySyscall/
 
 vmlinux-dirs	:= $(patsubst %/,%,$(filter %/, \
 		     $(core-y) $(core-m) $(drivers-y) $(drivers-m) \
diff -uprN -X linux-5.8.1/Documentation/dontdiff linux-5.8.1/mySyscall/Makefile linux-5.8.1Navneel.0360/mySyscall/Makefile
--- linux-5.8.1/mySyscall/Makefile	1970-01-01 05:30:00.000000000 +0530
+++ linux-5.8.1Navneel.0360/mySyscall/Makefile	2021-02-14 20:58:15.195405961 +0530
@@ -0,0 +1 @@
+obj-y := mySyscall.o
diff -uprN -X linux-5.8.1/Documentation/dontdiff linux-5.8.1/mySyscall/mySyscall.c linux-5.8.1Navneel.0360/mySyscall/mySyscall.c
--- linux-5.8.1/mySyscall/mySyscall.c	1970-01-01 05:30:00.000000000 +0530
+++ linux-5.8.1Navneel.0360/mySyscall/mySyscall.c	2021-02-14 21:39:18.489423332 +0530
@@ -0,0 +1,8 @@
+#include <linux/kernel.h>
+#include <linux/syscalls.h>
+
+SYSCALL_DEFINE0(mySyscall)
+{
+    printk("Hello world This is Navneel.0360\n");
+    return 0;
+}
Binary files linux-5.8.1/scripts/asn1_compiler and linux-5.8.1Navneel.0360/scripts/asn1_compiler differ
Binary files linux-5.8.1/scripts/extract-cert and linux-5.8.1Navneel.0360/scripts/extract-cert differ
Binary files linux-5.8.1/scripts/insert-sys-cert and linux-5.8.1Navneel.0360/scripts/insert-sys-cert differ
Binary files linux-5.8.1/scripts/selinux/mdp/mdp and linux-5.8.1Navneel.0360/scripts/selinux/mdp/mdp differ
Binary files linux-5.8.1/scripts/sign-file and linux-5.8.1Navneel.0360/scripts/sign-file differ
Binary files linux-5.8.1/scripts/sorttable and linux-5.8.1Navneel.0360/scripts/sorttable differ
Binary files linux-5.8.1/tools/objtool/objtool and linux-5.8.1Navneel.0360/tools/objtool/objtool differ
